sphinx_gallery/tests/test_scrapers.py

```diff
@@ -200,11 +200,14 @@ def test_figure_rst(ext):
 """
     assert image_rst == image_list_rst
 
-    # test issue #229
-    local_img = [os.path.join(os.getcwd(), "third." + ext)]
+
+def test_figure_rst_path():
```

Comment:
I think we still want this, right?
```suggestion
@pytest.mark.parametrize("ext", _KNOWN_IMG_EXTS)
def test_figure_rst_path():
```